sematic/api_client.py

```diff
@@ -382,7 +382,9 @@ def notify_graph_update(run_id: str):
 
 
 def _notify_event(namespace: str, event: str, payload: Any = None):
-    _post("/events/{}/{}".format(namespace, event), payload)
+    # we sent socket.io notifications to a dedicated endpoint, if it is configured
+    endpoint = f"{get_config().socket_io_url}/events/{namespace}/{event}"
```

Comment:
Missing `/api/v1/`

Reply:
It is included in the `get_config().socket_io_url` value.